app/frontend/src/components/CommunityGuidelines/CommunityGuidelines.tsx

```diff
@@ -83,22 +104,19 @@ export default function CommunityGuidelines({
         {error && <Alert severity="error">{error}</Alert>}
 
         <div className={classes.grid}>
-          {COMMUNITY_GUIDELINES.map((_, index) => (
+          {data.communityGuidelinesList.map((guideline, index) => (
             <React.Fragment key={index}>
-              <Avatar className={classes.avatar}>
-                <SvgIcon
-                  component={COMMUNITY_GUIDELINE_ICONS[index]}
-                  fontSize="large"
-                  className={classes.icon}
-                />
-              </Avatar>
+              <Avatar
+                className={classes.avatar}
+                src={`data:image/svg+xml,${encodeURIComponent(
+                  guideline.iconSvg
+                )}`}
+              />
               <div>
                 <Typography variant="h3" color="primary">
-                  {COMMUNITY_GUIDELINE_TITLES[index]}
-                </Typography>
-                <Typography variant="body1">
-                  {COMMUNITY_GUIDELINES[index]}
+                  {guideline.title}
                 </Typography>
+                <Typography variant="body1">{guideline.guideline}</Typography>
```

Comment:
Nitpick - seeing the same variable name repeated twice always feels a bit odd (reminds me of that whole `PagePage` thing that's still in our code 🙃 ). Maybe destructure the outer guideline object into `{ iconSvg, title, guideline }`, then you only need to `guideline` once here and likewise for other properties above without needing the `guideline.` prefix